boards/intel/adsp/Kconfig.intel_adsp

```diff
@@ -8,5 +8,5 @@ config BOARD_INTEL_ADSP
 	select SOC_INTEL_ACE15_MTPM if BOARD_INTEL_ADSP_ACE15_MTPM_SIM
 	select SOC_INTEL_ACE20_LNL if BOARD_INTEL_ADSP_ACE20_LNL
 	select SOC_INTEL_ACE20_LNL if BOARD_INTEL_ADSP_ACE20_LNL_SIM
-	select SOC_INTEL_ACE30_PTL if BOARD_INTEL_ADSP_ACE30_PTL
-	select SOC_INTEL_ACE30_PTL if BOARD_INTEL_ADSP_ACE30_PTL_SIM
+	select SOC_ACE30 if BOARD_INTEL_ADSP_ACE30_PTL
```

Comment:
Sorry was not clear, did not mean it needed to be done right now in this PR but just a note about fixing it in a later PR. All the socs will need fixing, I don't know the timeframe for when the feature for automatically generating symbols is planned however